java/README.md

```diff
@@ -1,19 +1,68 @@
 Welcome To The New Evolution In Sauce Labs Testing.
 The goal of Simple Sauce is to make your test automation and integration in Sauce Labs, simple :)
+Our fluent API helps you to understand all of the possible browser/OS combinations at design time.
+You no longer need to read extensive docs, just let your IDE guide you 😉
 
-Start a test on all defaults:
+##Quick Start: a test on all defaults:
 
 ```WebDriver driver = new SauceSession().start();```
 ```driver.findElement("id").click()```
 
-Run test on latest Mac/Safari:
+* Defaults are Windows 10 for non-Safari
+* latest Mac for Safari
+##Run a test on:
 
-```WebDriver driver = new SauceSession().withSafari().start();```
+###Latest Chrome
+```        
+SauceOptions options = new SauceOptions();
+options.withSafari();
+webDriver = new SauceSession(options).start();
+```
+###Latest Edge
+```        
+SauceOptions options = new SauceOptions();
+options.withEdge();
+webDriver = new SauceSession(options).start();
+```
+###Latest Firefox
+```        
+SauceOptions options = new SauceOptions();
+options.withFirefox();
+webDriver = new SauceSession(options).start();
+```
 
-Run test on Linux with Chrome:
+###Latest Safari
+```        
+SauceOptions options = new SauceOptions();
+options.withSafari();
+webDriver = new SauceSession(options).start();
+```
 
-```WebDriver driver = new SauceSession().withLinux().withChrome().start();```
-
-More complicated configuration:
-
-```WebDriver driver = new SauceSession().withLinux().withChrome("72.0").setIdleTimeout(100).start()```
+##If it's possible in Sauce, it's possible here:
+###Run test on Linux with Firefox:
+```        
+SauceOptions options = new SauceOptions().
+withLinux().
```

Comment:
Sauce doesn't support Linux outside of headless, so maybe we should not include this example?

Reply:
You appear to be correct. Ignore my comment.